GVRf/Framework/framework/src/main/jni/vulkan/vulkanCore.cpp

```diff
@@ -882,11 +883,11 @@ void VulkanCore::InitPipelineForRenderData(const GVR_VK_Vertices* m_vertices, Vu
 
     if(rdata->alpha_blend()) {
         att_state[0].blendEnable = VK_TRUE;
-        att_state[0].srcColorBlendFactor = VK_BLEND_FACTOR_ONE;
-        att_state[0].dstColorBlendFactor = VK_BLEND_FACTOR_ONE_MINUS_SRC_ALPHA;
+        att_state[0].srcColorBlendFactor = static_cast<VkBlendFactor>(m_conversionFlags.getFromGL(rdata->source_alpha_blend_func()));
```

Comment:
You need to add these parameters to hashCode of RenderData since we cache pipelines. If two objects have separate blends it will use pipeline created for the first object.